features/org.wso2.carbon.utils.feature/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>carbon-utils-parent</artifactId>
+        <groupId>org.wso2.carbon.utils</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.carbon.utils.feature</artifactId>
+    <packaging>carbon-feature</packaging>
+    <name>WSO2 Carbon Utils - Carbon Utils Feature</name>
```

Comment:
This name is not consistent with other maven modules in this. See below build summary.

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] WSO2 Carbon - Utils Parent ......................... SUCCESS [  4.901 s]
[INFO] WSO2 Carbon - Utils ................................ SUCCESS [ 11.640 s]
[INFO] WSO2 Carbon Utils - Carbon Utils Feature ........... SUCCESS [  2.032 s]
[INFO] ------------------------------------------------------------------------